doc/dev/BUILD.md

```diff
@@ -60,14 +60,26 @@ Installation can be done through CMake, by running the following command:
 cmake --install ${your_build_dir}
 ```
 
-Optional components can be installed by specifying the component name:
+Individual components can also be installed by specifying the component name:
 
 ```
 cmake --install ${your_build_dir} --component ${component_name}
 ```
 
-Here is the list of the optional components you can install:
-
-* `mimetypes`: Install plugins mimetype XML files for integration with Freedesktop (Linux only).
-* `configuration`: Install default configuration files , `config` and `thumbnail`.
-* `sdk`: Install the [libf3d](../libf3d/README_LIBF3D.md) SDK.
+Here is the list of all the components:
+
+Name|Installed by default|Operating system|Description
+------|------|------|------
+`application`|YES|ALL|F3D application
+`configuration`|NO|ALL|Default configuration files, `config` and `thumbnail`.
+`library`|YES|ALL|libf3d library binaries
+`plugin`|YES|ALL|libf3d plugins
+`dependencies`|NO|ALL|libf3d runtime dependencies (system libraries excluded)
+`sdk`|NO|ALL|libf3d SDK (headers and CMake config files)
+`licenses`|YES|ALL|F3D and third party licenses
+`documentation`|YES|Linux|`man` documentation
+`shellext`|YES|Windows/Linux|Desktop integration
```

Comment:
linux ?

Reply:
Yes, the Freedesktop integration is under this component